roles/custom/matrix-synapse-reverse-proxy-companion/templates/labels.j2

```diff
@@ -119,6 +119,44 @@ traefik.http.routers.matrix-synapse-reverse-proxy-companion-public-client-synaps
 ############################################################
 {% endif %}
 
+{% if matrix_synapse_reverse_proxy_companion_container_labels_internal_client_synapse_admin_api_enabled %}
+############################################################
+#                                                          #
+# Internal Synapse Admin API (/_synapse/client)            #
+#                                                          #
+############################################################
+
+traefik.http.routers.matrix-synapse-reverse-proxy-companion-internal-client-synapse-client-api.rule=PathPrefix(`/_synapse/client`)
+
+
+traefik.http.routers.matrix-synapse-reverse-proxy-companion-internal-client-synapse-client-api.service=matrix-synapse-reverse-proxy-companion-client-api
+traefik.http.routers.matrix-synapse-reverse-proxy-companion-internal-client-synapse-client-api.entrypoints=matrix-internal-matrix-client-api
```

Comment:
It'd be better if the entrypoints are coming from a variable (`matrix_synapse_reverse_proxy_companion_container_labels_internal_client_synapse_admin_api_traefik_entrypoints`), defined in `default/main.yml`  in the `matrix-synapse-reverse-proxy-companion` role.

Then, `group_vars/matrix_servers` can pass the correct internal entrypoint like this:

```yml
matrix_synapse_reverse_proxy_companion_container_labels_internal_client_synapse_admin_api_traefik_entrypo: "{{ matrix_playbook_internal_matrix_client_api_traefik_entrypoint_name }}"
```

Here's an example: https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/e000cbf0f42874bec05611d57a2922259da292bc/group_vars/matrix_servers#L4320